doc/source/using-ray-and-docker-on-a-cluster.md

```diff
@@ -1,236 +0,0 @@
-# Using Ray and Docker on a Cluster (Experimental)
```

Comment:
This should be subsumed by autoscaler (and also has a lot of information that seems to be out of scope for this project)